docs/app/import-export.md

```diff
@@ -0,0 +1,123 @@
+# Import / Export
+
+> The Content, User Directory, and File Library modules allow importing and exporting of multiple Items stored as files.
+> This makes it quick and easy for no-code Users to handle tasks like uploading and downloading new customer
+> information; pulling down sales data for transformation, reports, and analysis; and beyond.
+
+[[toc]]
+
+:::tip Before You Begin
+
+To use this utility, you will need to be familiar with [Collections](/getting-started/glossary/#collections),
+[Items](/getting-started/glossary/#items), and [Fields](/getting-started/glossary/#fields).
+
+:::
+
+:::tip Import / Export via the API
+
+This page details Importing and Exporting of Items as files via the no-code app. However, you can also
+[Import](/reference/system/utilities/#import-data-from-file) and
+[Export](/reference/system/utilities/#export-data-to-a-file) Items as files programmatically via the API.
+
+:::
+
+## Import Items
+
+<video autoplay muted loop controls title="Import Data From a File">
+	<source src="" type="video/mp4" />
+</video>
+
+To import Items from a file, follow the steps below.
+
+1. Navigate to the desired Module and Collection.\
+   Note: the User Directory and File Library are each composed of just one Collection.
+2. Click **"Import / Export"** in the Sidebar.
+3. Click into the import search box. A file browser will open.
+4. Select the desired file and click **"open"**.
+5. Click **"Start Import"** to import the Items.
+
+The new Items will now be in the Collection. The file itself will not be saved in the Directus Project.
+
+:::tip Importing CSV Files
+
+Any CSV column header that does not match a Field Name in the Collection will be ignored. If two columns in the CSV have
+the same header, then _(moving from left-to-right)_ the values from the second column will be used to create the new
+Items.
+
+:::
+
+:::tip Importing JSON Files
+
+Any value from a key that does not match a Field Name in the Collection will be ignored. If a JSON object has two keys
+with the same value, the second key-value pair _(moving from top-to-bottom)_ will be used to create the new Items.
```

Comment:
I think the same goes here. JSON with duplicate properties is very unusual, and not something Directus will export either 🤔 I think we can omit that to avoid unnecessary caution/confusion